### PR TITLE
capsule upgrade playbook naming change

### PR DESCRIPTION
**Pull request by pondrejk**
Working around the branding issue in 6.13 to keep capsule upgrade playbook tested

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_remoteexecution.py -k capsule

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_remoteexecution.py -k capsule

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_remoteexecution.py::test_positive_run_capsule_upgrade_playbook

**Review by Griffin-Sullivan — APPROVED** [review]
Ack pending test result

File: tests/foreman/api/test_remoteexecution.py [hunk 1]
```diff
@@ -36,14 +37,16 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
 
     :expectedresults: Capsule is upgraded successfully
 
+    :bz: 2152951
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_remoteexecution.py:40]:
```suggestion
    :BZ: 2152951
```

File: tests/foreman/api/test_remoteexecution.py [hunk 2]
```diff
@@ -36,14 +37,16 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
...
+
     :CaseImportance: Medium
     """
+    template_name = (
+        "Smart Proxy Upgrade Playbook" if is_open("BZ:2152951") else "Capsule Upgrade Playbook"
+    )
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_remoteexecution.py:46]:
```suggestion
    template_name = (
        'Smart Proxy Upgrade Playbook' if is_open('BZ:2152951') else 'Capsule Upgrade Playbook'
    )
```

File: tests/foreman/destructive/test_remoteexecution.py [hunk 3]
```diff
@@ -41,14 +42,16 @@ def test_negative_run_capsule_upgrade_playbook_on_satellite(target_sat):
 
     :expectedresults: Should fail
 
+    :bz: 2152951
+
     :CaseImportance: Medium
     """
+    template_name = (
+        "Smart Proxy Upgrade Playbook" if is_open("BZ:2152951") else "Capsule Upgrade Playbook"
+    )
```
Comment (Gauravtalreja1) [inline tests/foreman/destructive/test_remoteexecution.py:51]:
```suggestion
    template_name = (
        'Smart Proxy Upgrade Playbook' if is_open('BZ:2152951') else 'Capsule Upgrade Playbook'
    )
```

File: tests/foreman/api/test_remoteexecution.py [hunk 4]
```diff
@@ -36,14 +37,16 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
...
     template_id = (
-        target_sat.api.JobTemplate()
-        .search(query={'search': 'name="Capsule Upgrade Playbook"'})[0]
-        .id
+        target_sat.api.JobTemplate().search(query={'search': f'name="{template_name}"'})[0].id
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_remoteexecution.py:48]:
```suggestion
        target_sat.api.JobTemplate().search(query={'search': 'name=f"{template_name}"'})[0].id
```

**Review by Gauravtalreja1 — COMMENTED** [review]
LGTM, added a few suggestions, and PRT will fail due to https://bugzilla.redhat.com/show_bug.cgi?id=2154397

**Comment by Satellite-QE** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_remoteexecution.py -k capsule

**Pull request merged**